lib/java/com/google/android/material/textfield/TextInputLayout.java

```diff
@@ -718,17 +716,6 @@ public TextInputLayout(@NonNull Context context, @Nullable AttributeSet attrs, i
     setHelperText(helperText);
   }
 
-  @Override
-  public void onGlobalLayout() {
```

Comment:
The OnGlobalLayoutListener was originally added in https://github.com/material-components/material-components-android/commit/93360a5a5d2549c14599507db94423906d5a9757 in order to prevent an onMeasure() infinite loop bug, wouldn't removing OnGlobalLayoutListener reintroduce that bug?

Reply:
I don't know about the bug, but the situation from the explanation shouldn't happen since `endLayout.measure()` for the visible `endLayout` is guaranteed to be called by `super.onMeasure()`, which is before `updateDummyDrawables()`.